compiling issue with mysql positive grammar

It seems that the NULL in the grammar is conflicted with null in std lib

```
mkdir -p output
mkdir -p generated
/usr/bin/java -jar antlr-4.9.2-complete.jar -Dlanguage=Cpp -listener -visitor -o generated -package parsers MySQLLexer.g4 MySQLParser.g4
warning(125): MySQLParser.g4:1510:12: implicit definition of token ADMIN in parser
warning(125): MySQLParser.g4:2304:11: implicit definition of token NULL in parser
warning(125): MySQLParser.g4:2507:6: implicit definition of token OPTIONAL in parser
warning(154): MySQLParser.g4:1355:0: rule blockStatement contains an optional block with at least one alternative that can match an empty string
g++ -c -I runtime-osx/antlr4-runtime/ -I generated -std=c++11    hello.cpp  -o output/hello.o
In file included from hello.cpp:7:
generated/MySQLParser.h:259:19: error: expected identifier
    ADMIN = 1093, NULL = 1094, OPTIONAL = 1095
                  ^
/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/stddef.h:84:18: note: expanded from macro 'NULL'
#    define NULL __null
                 ^
In file included from hello.cpp:7:
generated/MySQLParser.h:10825:47: error: expected member name or ';' after declaration specifiers
    std::vector<antlr4::tree::TerminalNode *> NULL();
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/stddef.h:84:18: note: expanded from macro 'NULL'
#    define NULL __null
                 ^
In file included from hello.cpp:7:
generated/MySQLParser.h:10826:33: error: expected member name or ';' after declaration specifiers
    antlr4::tree::TerminalNode* NULL(size_t i);
    ~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/stddef.h:84:18: note: expanded from macro 'NULL'
#    define NULL __null
                 ^
hello.cpp:19:3: error: unknown type name 'ExprLexer'
                ExprLexer lexer(&input);
                ^
hello.cpp:28:3: error: unknown type name 'ExprParser'
                ExprParser parser(&tokens);
                ^
5 errors generated.
make: *** [hello] Error 1
```

--- a/sql/mysql/Positive-Technologies/MySqlParser.g4
+++ b/sql/mysql/Positive-Technologies/MySqlParser.g4
@@ -2304,8 +2304,8 @@ specificFunction
       '(' expression
        ',' expression
          (RETURNING convertedDataType)?
-         ((NULL | ERROR | (DEFAULT defaultValue)) ON EMPTY)?
-         ((NULL | ERROR | (DEFAULT defaultValue)) ON ERROR)?
+         ((NULL_LITERAL | ERROR | (DEFAULT defaultValue)) ON EMPTY)?
+         ((NULL_LITERAL | ERROR | (DEFAULT defaultValue)) ON ERROR)?
        ')'                                                          #jsonValueFunctionCall
     ;
 

